src/validateData.js

```diff
@@ -0,0 +1,50 @@
+const validateData = (textToValidate, toCase) => {
+  const validCases = [
+    'SNAKE',
+    'KEBAB',
+    'CAMEL',
+    'PASCAL',
+    'UPPER',
+  ];
+  const errorMessages = {
+    noText: 'Text to convert is required.'
+    + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    noToCase: '"toCase" query param is required.'
+    + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    notValidCase: 'This case is not supported.'
+    + ' Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+  };
```

Comment:
Create these variables outside you the `validateData` function to avoid re-creating whenever the function executed